tests/common.go

```diff
@@ -12170,3 +12170,128 @@ func WaitForVolumeClean(vol *volume.Volume) error {
 	_, err := task.DoRetryWithTimeout(t, 30*time.Minute, 60*time.Second)
 	return err
 }
+
+// GetFADetailsUsed Returns list of FlashArrays used in the cluster
+func GetFADetailsUsed() ([]pureutils.FlashArrayEntry, error) {
+	//get the flash array details
+	volDriverNamespace, err := Inst().V.GetVolumeDriverNamespace()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get details on FlashArray used in the cluster")
+	}
+
+	pxPureSecret, err := pureutils.GetPXPureSecret(volDriverNamespace)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to get Px Pure Secret")
+	}
+
+	if len(pxPureSecret.Arrays) > 0 {
+		return pxPureSecret.Arrays, nil
+	}
+	return nil, fmt.Errorf("Failed to list FA Arrays ")
+}
+
+func FlashArrayGetIscsiPorts() (map[string][]string, error) {
```

Comment:
can you move all these to pureutils as these are specifc to FA/FB

Reply:
sure , will move it to pure utils

Reply:
@lsrinivas-pure , this is the common function that is derived from functions listed in pure utils , I guess this should be in common utils . 
will be using this functions on multiple scripts belongs to FA going forward